packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -148,4 +151,136 @@ export class AiAgentModel {
 
         return agent;
     }
+
+    async createAgent(
+        args: Pick<
+            ApiCreateAiAgent,
+            'name' | 'projectUuid' | 'tags' | 'integrations'
+        > & {
+            organizationUuid: string;
+        },
+    ): Promise<AiAgent> {
+        const [agent] = await this.database(AiAgentTableName)
+            .insert({
+                name: args.name,
+                project_uuid: args.projectUuid,
+                organization_uuid: args.organizationUuid,
+                tags: args.tags,
+                description: null,
+                image_url: null,
+            })
+            .returning('*');
+
+        const insertedIntegrations: ApiCreateAiAgent['integrations'] = [];
+
+        if (args.integrations && args.integrations.length > 0) {
+            const integrationPromises = args.integrations.map(
```

Comment:
this is not required, you can just do args.integrations?.map

Reply:
done 👍